pkg/controller/volume/expand/expand_controller.go

```diff
@@ -224,7 +226,7 @@ func (expc *expandController) syncHandler(key string) error {
 		return err
 	}
 
-	pv, err := getPersistentVolume(pvc, expc.pvLister)
+	pv, err := expc.getPersistentVolume(pvc)
```

Comment:
Did you consider adding a PV to `expc.pvLister` after the PV is saved with the new size? Would that save us from an API call?

Reply:
I did. it is kind of complicated to do that because expansion call is handled by operation_generator(and we don't have informer there) and it will require some larger re-factoring to do that correctly. I think overall since this call is only made after it has been determined expansion is needed and we don't expect people to be resizing volumes that often, it is I think acceptable tradeoff.

Reply:
Ack